### PR TITLE
Adds Kyrgyz translation to web basics workshop.

### PR DESCRIPTION
**Pull request by disaparb**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -51,6 +51,11 @@ weight = 7
 languageName = "繁體中文"
 contentDir = "content/traditional-chinese"
 
+[Languages.ky]
```
Comment (ozhang22) [inline config.toml:54]: Thank you for adding this!

**Review by ozhang22 — APPROVED** [review]
Looks good, but please get someone who is also fluent to do a quick review.

**Review by aminueza — APPROVED** [review]

File: content/kyrgyz/web-basics/answer-key.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+### Эскертүү:
+
+Бул окуу программасы студенттердин чыгармачылык менен иштөөсүнө шарт түзөт. Төмөндөгүлөр алар эмне кыла аларынын бир мисалы гана , эгер керек болсо, өзүңүздүн жеке ойуңузду колдонуңуз жана кеңеш сураңыз. Бул жерде берилген жооп ачкычы кыйла узун, анткени биз контекст үчүн толук HTML үзүндүлөрүн кошуп жатабыз.
+
+Студент 'Өзүңүз байкап көрүңүз!' баскычын басканда HTML кодунун айрымдары (жана CSS коду) менен жаңы өтмөк ачылат. 
+
+## Сынак 1
+
+Чечимде:
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/answer-key.md:17]: Suggested text change: 
**Current:** Чечимде:
**Suggested:** Чечимде төмөнкүлөр болушу керек:

File: content/kyrgyz/web-basics/answer-key.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    <li>Үндүк</li>
+    <li>Жер жаңгак майы</li>
+    <li>Жип сыры</li>
+  </ul>
+</body>
+```
+
+## Сынак 2
+
+Чечимде:
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/answer-key.md:43]: Suggested text change: 
**Current:** Чечимде:
**Suggested:** Чечимде төмөнкүлөр болушу керек:

File: content/kyrgyz/web-basics/answer-key.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+1. 'embed' баскычын басыңыз жана GIF кыстаруу кодун көчүрүңүз.
+2. GIF кыстаруу кодун ```<body>``` тегинин ичине коюңуз.
+
+Мисал iframe коду:
+
+```HTML
+<iframe src="https://giphy.com/embed/oywrR92C9YBlrSuUKP" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/nuevofoundation-rockstar-nuvi-oywrR92C9YBlrSuUKP">via GIPHY</a></p>
+```
+
+'Код өзүңүз байкап көрүңүз!' үлгүсүнө интеграцияланган:
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/answer-key.md:103]:
```suggestion
Код 'Өзүңүз байкап көрүңүз!' үлгүсүнө интеграцияланган:
```

File: content/kyrgyz/web-basics/answer-key.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+        <li>Мен сен каалабаган жержаңгак майын жейм</li>
+        <li>Мен сени тыйын чычкандардан коргойм</li>
+      </ul>
+    </div>
+
+   <!-- Сынак 4: Байланыш формасы -->
+    <div class="contact-section">
+      <h3>Мени менен байланыш</h3> 
+      <hr>
+      <!-- Change the following line -->
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/answer-key.md:171]: translation needed

File: content/kyrgyz/web-basics/basics.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Негиздер"
+date: 2019-09-03T17:53:41-07:00
+draft: false
+weight: 2
+---
+
+## Веб-сайт деген эмне?
+
+Веб-сайт - бул Интернетте табыла турган баракча. Дарек тилкесинде даректи (ака уникалдуу ысым) тергениңизде, ал браузер аркылуу көрсөтүлөт.
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/basics.md:10]:
```suggestion
Веб-сайт - бул Интернеттеги баракча. Дарек тилкесинде даректи (б.а. уникалдуу ысым) тергениңизде, ал браузер аркылуу көрсөтүлөт.
```

File: content/kyrgyz/web-basics/conclusion.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Эгер сизге кызык болсо"
+date: 2019-09-03T17:53:41-07:00
+draft: false
+weight: 12
+---
+
+![alt text](../media/benji-and-me-sm.png "Бенжи экөөбүздүн сүрөтүбүз")
+
+Бенжи сизди тынчсыздандырышы мүмкүн, бирок биз сизге анын үйүн сиз куруп жаткан веб-сайт сыяктуу веб-сайт аркылуу тапканын айтууга кубанычтабыз. Бенжи азыр күчүк тамак алуу үчүн жаңы амалдарды үйрөнүп жаткан сүйүктүү уй бүлө мүчөсү.
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/conclusion.md:10]:
```suggestion
Бенжи сизди тынчсыздандырышы мүмкүн, бирок биз сизге анын үйүн сиз куруп жаткандай бир веб-сайт аркылуу тапканын айтууга кубанычтабыз. Бенжи азыр күчүк, тамак алуу үчүн жаңы амалдарды үйрөнүп жаткан сүйүктүү уй бүлө мүчөсү.
```

File: content/kyrgyz/web-basics/hosting.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Вебсайтты интернетте жайгаштыруу"
+date: 2019-09-03T17:53:41-07:00
+draft: false
+weight: 11
+---
+
+## Сайтыңызды интернетте жайгаштырыңыз!
+
+Эми сиз өзүңүздүн жеке веб-сайтыңызды жасагандан кийин, келиңиз, аны интернетке жайгаштыралы. Андан кийин баары ага кире алышат! Бул үчүн сизге GitHub каттоо эсеби керек болот. Эгер сизде жок болсо, аны түзүңүз [бул жерде](https://github.com/).
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/hosting.md:10]:
```suggestion
Эми сиз өзүңүздүн жеке веб-сайтыңызды жасагандан кийин, келиңиз, аны интернетке жайгаштыралы. Андан кийин баары ага кире алышат! Бул үчүн сизге GitHub каттоо эсеби керек болот. Эгер сизде жок болсо, каттоодон өтүңүз [бул жерде](https://github.com/).
```

File: content/kyrgyz/web-basics/hosting.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+Каттоо эсебиңизди түзүп, электрондук почта дарегиңизди ырастагандан кийин, сиз бул баракка түшөсүз.
+
+<p style="text-align: center; "><img src="../media/newAccount.png" alt="githubдагы жаңы аккаунттун ачылыш бети" width="1200"/></p>
+
+Эми сиз сайтыңыздын контентти сактоо үчүн жаңы репозиторий түзүшүңүз керек болот! Репозиторийди бир долбоор үчүн көптөгөн файлдарды жана папкаларды сактай ала турган чоң папка деп ойлоңуз. Сиз `Start a Project` же `Create a new repository` тандап менен бирөөнү түзө аласыз. Эки маанилүү нерсени эстен чыгарбоо керек:
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/hosting.md:16]:
```suggestion
Эми сиз сайтыңыздын контентти сактоо үчүн жаңы репозиторий түзүшүңүз керек болот! Репозиторийди бир долбоор үчүн көптөгөн файлдарды жана папкаларды сактай ала турган чоң папка деп ойлоңуз. Сиз `Start a Project` же `Create a new repository` тандoo менен бирөөнү түзө аласыз. Эки маанилүү нерсени эстен чыгарбоо керек:
```

File: content/kyrgyz/web-basics/hosting.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Репоңузду <b>эч кандай боштуктар жок атаңыз</b>. Бирок боштуктардын ордна сызыкчаларды же баш тамгаларды колдонсоңуз болот!
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/hosting.md:18]:
```suggestion
1. Репоңузду <b>эч кандай боштуктар жок атаңыз</b>. Бирок боштуктардын ордуна сызыкчаларды же баш тамгаларды колдонсоңуз болот!
```

File: content/kyrgyz/web-basics/media-elements.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+![alt text](../media/flipbook.gif "флипбук gif")
+
+---
+
+## Видео кош
+
+Тексттен жана сүрөттөрдөн тышкары, биз веб-сайттарга кошкон эң көп нерселердин бири бул видеолор. Видео файлдары абдан чоң, андыктан YouTube сыяктуу видеону жүктөө жана көрсөтүү үчүн кызматты колдонгонуңуз жакшы. YouTube'га жүктөлгөндөн кийин, аны кантип веб-сайтыбызда ойнота алабыз? Биз <b>iframe</b> деп аталган атайын тегди колдонобуз.
+
+Iframes биздин сайтын ичинде башка веб-сайттардан HTMLди көрсөтөт. Бул терезе сыяктуу, анда сиз учурдагы веб-сайттын ичинен интернетте сыртта эмне болуп жатканын көрө аласыз. Бул процесс кыстаруу деп аталат.
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/media-elements.md:20]:
```suggestion
Iframes биздин сайттын ичинде башка веб-сайттардан HTMLди көрсөтөт. Бул терезе сыяктуу, анда сиз учурдагы веб-сайттын ичинен интернетте сыртта эмне болуп жатканын көрө аласыз. Бул процесс embedding деп аталат.
```

File: content/kyrgyz/web-basics/media-elements.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+Көптөгөн сайттар веб-сайтыңызга кыстаруу үчүн iframes сунуштайт. Мисал катары, бул жерде сиз YouTube видеосу үчүн iframe кодун кантип табасыз:
+
+![alt text](../media/get-iframe.gif "iframes менен кыстаруу мисалы")
+
+### iframe үлгүсү
+Төмөнкү кодду көчүрүңүз:
+`<iframe width="560" height="252" src="https://www.youtube.com/embed/lQsqEBSGdyc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen> </iframe>`
+
+Кодду Code Penге киргизиңиз жана ал ушундай болот:
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/media-elements.md:34]:
```suggestion
Кодду Code Penге киргизсеңиз ал ушундай көрүнөт:
```

File: content/kyrgyz/web-basics/putting-together.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+Көпчүлүк тегдерге ачылыш жана жабуу теги керек. Мисалы, сиз бардык барактардын жогору жагында `<html>` ачуу теги жана аягында жабылуучу `</html>` теги бар экенин көрөсүз. Бул браузерге HTML элементтери кайда башталып, кайда бүтөрүн айтып берет. Тырмакча сыяктуу тегдерди ойлонуңуз; мисалы, компьютер `<html>` жана `</html>` тегдердин ортосундагы бардыгы HTML веб-сайтынын бир бөлүгү экенин түшүнөт.
+
+Ошондой эле бир нече тегдер бар, мисалы `<img>` сүрөт теги, алар өз алдынча жабылат, башкача айтканда, аларга ачуу теги гана керек жана жабуу теги керек эмес.
+
+Бул жерде бир мисал:
+
+ <p class="codepen" data-height="500" data-theme-id="dark" data-default-tab="html,result" data-user="Sunny-Dee" data-slug-hash="ErRraG" style="height: 500px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="HTML Basics">
+    <span>See the Pen <a href="https://codepen.io/Sunny-Dee/pen/ErRraG/">HTML негиздери</a> Делиана Эскобариники (<a href="https://codepen.io/Sunny-Dee">@Sunny-Dee</a>)
```
Comment (ikrmkg) [inline content/kyrgyz/web-basics/putting-together.md:23]: Suggested text change:
Current: See the Pen [HTML негиздери] Делиана Эскобариники ([@Sunny-Dee]) on [CodePen].
Suggested: CodePen'деги Делиана Эскобардын(@sunnydee) Pen [HTML негиздери]'н көрүңүз

File: themes/docdock/i18n/ky.toml [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+# next-prev-button.html
+[up-next]
+other = "Кийинки:"
+
+[previous]
+other = "Мурунку:"
+
+# custom-footer.html
+[join-social-media]
+other = "Социалдык тармактардагы кошулуңуз."
```
Comment (ikrmkg) [inline themes/docdock/i18n/ky.toml:14]:
```suggestion
other = "Социалдык тармактардагы сүйлөшүүлөргө кошулуңуз."
```

File: content/kyrgyz/web-basics/activity-2.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+
+Төмөндөгү мисалдарды караңыз:
+<p class="codepen" data-height="426" data-theme-id="dark" data-default-tab="css,result" data-user="Sunny-Dee" data-slug-hash="qgvpQE" style="height: 426px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="Styling it up"> Pen бул жерде <a href="https://codepen.io/Sunny-Dee/pen/qgvpQE/"> Стилдөө </a> Делиана Эскобардыкы (<a href="https://codepen.io/Sunny-Dee">@Sunny-Dee</a>) <a href="https://codepen.io">CodePen</a>.</p>
+<script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+<b>HTML кодун көрүү үчүн жогорку сол бурчтагы "HTML" өтмөгүн, ал эми CSS кодун көрүү үчүн "CSS" өтмөгүн басыңыз.</b>
+
+Келгиле, HTML кодун карап көрөлү. &lt;div&gt; <code>"info"</code> атрибуту менен тегти таап көрүңүз. Ал буга <code>&lt;div id="info"&gt;</code> окшош. Бул div-ге id берип, биз андан кийин ушул элемент үчүн стилди көрсөтүүгө CSS-ти колдоно алабыз.
```
[the remark this answers is not in the document]
Comment (disaparb) [reply content/kyrgyz/web-basics/activity-2.md:23]: done

**Comment by disaparb** [comment]
@ikrmkg thanks for reviewing! Pushed new changes. Please approve.

**Pull request merged**